database/migrations/2019_02_09_200203_add_gender_type.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use App\Models\User\User;
+use App\Models\Contact\Gender;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddGenderType extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('genders', function (Blueprint $table) {
+            $table->string('type')->after('name')->nullable();
+        });
+
+        $woman = null;
+        $man = null;
+        $other = null;
+        $woman_en = trans('app.gender_female', [], 'en');
```

Comment:
Here again, choose either camelCase or `_`, but not a mix of both.
Therefore the variable should probably be called `womanEn` or something.

Reply:
thanks